packages/editor/src/hooks/custom-sources-backwards-compatibility.js

```diff
@@ -70,18 +70,17 @@ const createWithMetaAttributeSource = ( metaAttributes ) =>
 						attributes={ mergedAttributes }
 						setAttributes={ ( nextAttributes ) => {
 							const nextMeta = Object.fromEntries(
-								Object.entries(
-									// Filter to intersection of keys between the updated
-									// attributes and those with an associated meta key.
-									pickBy(
-										nextAttributes,
-										( value, key ) => metaAttributes[ key ]
+								Object.entries( nextAttributes ?? {} )
```

Comment:
`nextAttributes` could theoretically be nullish, so we're falling back to an empty object just in case.